### DISCUSSION TITLE
Feature bounties?

### DISCUSSION THREAD

**Discussion by bmfurtado** [nikitabobko/AeroSpace#746] [general]
I suppose this is mostly a question for @nikitabobko... I was wondering if you had considered having a system for feature bounties to help you fund the development of new features.

For instance, I've been using (and loving) AeroSpace for a while now, but one of the challenges I keep hitting is that the experience when using a very wide screen (such as 32:9) is weird when few windows are open. I've seen a few of proposals in this area and, while I don't have enough free time to help with the project, I would value seeing improvements in this area enough to help fund its development.

**Comment by nikitabobko** [comment]
It's not only about money but also about available free time and personal preference in features.

Thanks for the suggestion, but right now, I don't consider feature bounties, because they won't make a difference for me.
